pom.xml

```diff
@@ -0,0 +1,621 @@
+      <name>closure-templates-discuss</name>
+      <subscribe>closure-templates-discuss+subscribe@googlegroups.com</subscribe>
+      <unsubscribe>closure-templates-discuss+unsubscribe@googlegroups.com</unsubscribe>
+      <post>closure-templates-discuss@googlegroups.com</post>
+      <archive>https://groups.google.com/forum/#!forum/closure-templates-discuss</archive>
+    </mailingList>
+  </mailingLists>
+
+  <scm>
+    <connection>scm:git:https://github.com/google/closure-templates.git</connection>
+    <developerConnection>scm:git:git@github.com:google/closure-templates.git</developerConnection>
+    <url>https://github.com/google/closure-templates.git</url>
+    <tag>HEAD</tag>
+  </scm>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <java.version>1.7</java.version>
+    <guice.version>3.0</guice.version>
+    <soy.examples>${project.basedir}/examples</soy.examples>
+    <soy.examples.out>target/examples</soy.examples.out>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>javax.inject</groupId>
+      <artifactId>javax.inject</artifactId>
+      <version>1</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.inject</groupId>
+      <artifactId>guice</artifactId>
+      <version>${guice.version}</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.inject.extensions</groupId>
+      <artifactId>guice-assistedinject</artifactId>
+      <version>${guice.version}</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.inject.extensions</groupId>
+      <artifactId>guice-multibindings</artifactId>
+      <version>${guice.version}</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>14.0</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.ibm.icu</groupId>
+      <artifactId>icu4j</artifactId>
+      <version>51.1</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>aopalliance</groupId>
+      <artifactId>aopalliance</artifactId>
+      <version>1.0</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>args4j</groupId>
+      <artifactId>args4j</artifactId>
+      <version>2.0.23</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.code.findbugs</groupId>
+      <artifactId>jsr305</artifactId>
+      <version>2.0.3</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.apache.ant</groupId>
+      <artifactId>ant</artifactId>
+      <version>1.8.3</version>
+      <scope>provided</scope>
```

Comment:
The `provided` scope keeps ant out of the `with-dependencies` artifact, the shaded artifacts and as transitive dependency on soy. With `optional` only, ant gets packaged in `with-dependencies` and shaded artifacts. I am not sure a case where a user would opt to pull in ant with soy, if there is a reason i can add the `optional`.